### PR TITLE
Update Foundation, AEM and Cortex Models versions

### PR DESCRIPTION
**Pull request by davidcunado-arm**
Trusted Firmware has been tested as part of its CI system against Cortex
and Foundation models in the 11.0 Model release available on
developer.arm.com. Trusted Firmware has also been tested against the v8.5
AEM model.

This patch updates the user guide documentation to reflect the version of
the Foundation, AEM and Cortex Models that Trusted Firmware has been
tested against.

Change-Id: I3b5b4d1e4220bda1dcc88aa9cfa01fa711ed92cd
Signed-off-by: David Cunado <david.cunado@arm.com>

### CODE REVIEW

File: docs/user-guide.md [hunk 1]
```diff
@@ -1189,15 +1189,24 @@ to load the ELF file over JTAG on Juno.
 The latest version of the AArch64 build of ARM Trusted Firmware has been tested
 on the following ARM FVPs (64-bit host machine only).
 
-*   `Foundation_Platform` (Version 10.2, Build 10.2.20)
-*   `FVP_Base_AEMv8A-AEMv8A` (Version 8.4, Build 0.8.8402)
-*   `FVP_Base_Cortex-A57x4-A53x4` (Version 8.4, Build 0.8.8402)
+NOTE: Unless otherwise stated, the model version is Version 11.0 Build 34.
```
Comment (danh-arm) [inline docs/user-guide.md]: Please keep the full build number (presumably 11.0.34) as this is what is reported by launching the FVP with --version (see note below).

File: docs/user-guide.md [hunk 2]
```diff
@@ -1189,15 +1189,24 @@ to load the ELF file over JTAG on Juno.
...
+
+*   `Foundation_Platform`
+*   `FVP_Base_AEMv8A-AEMv8A` (Version 8.5, Build 8502)
```
Comment (danh-arm) [inline docs/user-guide.md]: Please keep the full build number (presumably 0.8.8502) as this is what is reported by launching the FVP with --version (see note below). Also, this make it clearer which FVP codeline this comes from.

File: docs/user-guide.md [hunk 3]
```diff
@@ -1189,15 +1189,24 @@ to load the ELF file over JTAG on Juno.
...
+*   `FVP_Base_Cortex-A72x4`
+*   `FVP_Base_Cortex-A73x4-A53x4`
+*   `FVP_Base_Cortex-A73x4`
 
 The latest version of the AArch32 build of ARM Trusted Firmware has been tested
 on the following ARM FVPs (64-bit host machine only).
 
-*   `FVP_Base_AEMv8A-AEMv8A` (Version 8.4, Build 0.8.8402)
-*   `FVP_Base_Cortex-A32x4` (Version 10.1, Build 10.1.32)
+*   `FVP_Base_AEMv8A-AEMv8A` (Version 8.5, Build 8502)
```
Comment (danh-arm) [inline docs/user-guide.md]: Please keep the full build number

Comment (davidcunado-arm) [inline docs/user-guide.md]: Updated to full build number.

**Comment by danh-arm** [comment]
This PR needs rebasing to use reSt format now that https://github.com/ARM-software/arm-trusted-firmware/pull/1011 is merged

**Pull request merged**